pwa/src/components/translations/tableNamesTable.tsx

```diff
@@ -96,10 +96,10 @@ export default function TableNamesTable() {
                         headerName: " ",
                         renderCell: (tables: { name: string }) => {
                           return (
-                            <Link className="utrecht-link d-flex justify-content-end" to={`/translations/${tables.name}`}>
-                              <button className="utrecht-button btn-sm btn-success">
-                                <i className="fas fa-edit pr-1" />
-                                Edit
+                            <Link className="utrecht-link d-flex justify-content-end" to={`/translation-tables/${tables.name}`}>
```

Comment:
I think we should refactor this to `/translations-tables/id`. For the following reasons:

1. To be more consistent (we are currently using the `id` for every route in very resource;
2. To be more precise (e.g. right now, when you have a translation _table_ called `new` we will go to the create instead of the edit)